cea/optimization/optimization_main.py

```diff
@@ -62,55 +61,23 @@ def moo_optimization(locator, weather_file, gv, config):
     # read total demand file and names and number of all buildings
     total_demand = pd.read_csv(locator.get_total_demand())
     building_names = total_demand.Name.values
-    gv.num_tot_buildings = total_demand.Name.count()
-    lca = LcaCalculations(locator, config.detailed_electricity_pricing)
+    lca = LcaCalculations(locator, config.optimization.detailed_electricity_pricing)
     prices = Prices(locator, config)
 
     # pre-process information regarding resources and technologies (they are treated before the optimization)
     # optimize best systems for every individual building (they will compete against a district distribution solution)
     print("PRE-PROCESSING")
-    solar_features = preproccessing(locator, total_demand, building_names, weather_file, gv, config, prices, lca)
+    preproccessing(locator, total_demand, weather_file, config)
 
-    # optimize the distribution and linearize the results(at the moment, there is only a linearization of values in Zug)
+    # optimize the distribution and linearize the results
     print("NETWORK OPTIMIZATION")
     network_features = NetworkOptimizationFeatures(config, locator)
 
     # optimize conversion systems
-    print("CONVERSION AND STORAGE OPTIMIZATION")
-    master_main.non_dominated_sorting_genetic_algorithm(locator, building_names, solar_features,
-                                                        network_features, gv, config, prices, lca)
-
-    print("GETTING EXTRA COSTS AND EMISSIONS FROM ELECTRICITY")
-    extra_costs_emissions_primary_energy(building_names, config, lca, locator, prices, total_demand)
-
```

Comment:
just to clarify, is this part `extra_costs_emissions_primary_energy` redundant?
if yes, the corresponding scripts that are called in the function should be deleted as well

Reply:
is it? it's hard to say. it _might_ be changing one of its parameters. this is why changing your parameters is considered something to avoid (or make very explicit in the function name)

Reply:
yes it is extra, I am removing this